OpenTrons/opentrons-api
# refactor(app): Display pipette offsets in calibration card

Add information displaying the pipette offset calibration for currently attached pipettes (per [the design](https://www.figma.com/file/nbDHskbW5zaXwtrLvEIKVj/calibration-overhaul?node-id=124%3A0)) to the calibration card.

This has a couple other things that were required:
- Expose the `tiprackUri` component of the pipette offset calibration via HTTP; it was previously saved but not exposed
- Fetch pipettes and pipette offset calibrations when the calibration page is displayed
- New selector to fuse attached pipettes and offset calibrations

## Risk
None, behind feature flag

## Review Requests

- Is this the right way / place to make the new selector?
- Is this the right way / place to fetch data?
- General code review

## Testing
- Make sure you put the robot server build from this PR on your robot and turn on the calibration overhaul flags on both app  and robot
- stop the API server and restart the app.
   - Look at the robot page for the unhealthy robot and see that there are no pipettes connected
   - Restart the api server while continuing to look at the robot page; when the api server comes up, the robot page should refresh with attached pipettes and calibration on its own
- check that the proper data for
  - attached and calibrated
  - not attached 
  - attached and not calibrated
   work
- check that the right tiprack is displayed. In theory this should work with a custom tiprack.


Closes #6612
